src/scripts/selectors.js

```diff
@@ -42,6 +42,16 @@ export const formatAmount = (props) => {
         }
     )
 }
+export const estimatedHours = (props) => {
```

Comment:
Rename this to `estimateExpectedHours` or `getExpectedHours` or `calculateExpectedHours` to [utilize parts of speech](https://github.com/setlife-network/trinary/blob/develop/CONTRIBUTING.md#5-utilize-parts-of-speech)